driftbase/api/parties.py

```diff
@@ -37,6 +37,7 @@ class PartyGetRequestSchema(ma.Schema):
 
 class PartyInvitesSchema(ma.Schema):
     inviter_id = ma.fields.Integer()
+    leave_existing_party = ma.fields.Bool(required=False, dump_default=False, metadata=dict(description="Whether or not to leave an existing party before joining the new one"))
```

Comment:
I'm 99% sure that `load_default` is what I want here since the `required` kwarg is only used in `_validate_missing` and that is only ever called in the `deserialize` function.

Will update the PR.